packages/ruby/spec/readme/payload_spec.rb

```diff
@@ -9,24 +11,28 @@
   let(:ip_address) { Socket.ip_address_list.detect(&:ipv4_private?).ip_address }
 
   it 'returns JSON matching the payload schema' do
+    id = '1'
     result = described_class.new(
       har,
-      { id: '1', label: 'Owlbert', email: 'owlbert@example.com' },
+      { id: id, label: 'Owlbert', email: 'owlbert@example.com' },
       ip_address,
       development: true
     )
 
+    expect(JSON.parse(result.to_json)["group"]["id"]).to match(Readme::Mask.mask(id))
```

Comment:
This is so dumb, but the request object we create _only_ exposes a `to_json` method. So the only way to get the headers is to serialize the request to JSON... and then parse the JSON. 🙃